libtransmission/rpc-server.cc

```diff
@@ -77,39 +77,101 @@ auto inline constexpr TrUnixAddrStrLen = size_t{ sizeof(((struct sockaddr_un*)nu
 
 enum tr_rpc_address_type
 {
-    TR_RPC_AF_INET,
-    TR_RPC_AF_INET6,
-    TR_RPC_AF_UNIX
+    TR_RPC_UNIX_ADDR,
+    TR_RPC_INET_ADDR,
```

Comment:
This changes the order s.t. any zero-initialized struct that holds a `tr_rpc_address_type` will now default to a unix addr instead of an inet addr. Might be better to preserve the old order by putting `TR_RPC_INET_ADDR` first in the list